### ISSUE TITLE
MonthCalendar control - Narrator is not announcing the date when user is navigating on it by using right and left arrow keys

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#1910]
**Repro Steps:​**
1. Launch .NET Core WinForms app with MonthCalendar control.
2. Start Narrator.
4. Verify that narrator is announcing the date or not when users is navigating in the month calendar control using right and left arrow key

**Actual:**
Narrator is not announcing the date when user is navigating on it using right and left arrow key. Narrator is announcing only 'Month Calendar'.

**Expected:**
Narrator should announce the date when user is navigating on it by using right and left arrow key.

**User Impact:**
Screen reader users will face difficulty understand the date when navigating month calendar control elements.

**Comment by M-Lipin** [comment]
DevDiv issue: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/956425

**Comment by M-Lipin** [comment]
Resolving this issue as the fixing PR #2090 is merged to release/3.1.

**Comment by Zheng-Li01** [comment]
Verified with latest .NET Core 3.1 3.1.100-GA-014700 build, this issue is fixed
![GitHubIssue](https://user-images.githubusercontent.com/38325459/68742926-41c55300-062c-11ea-8ca8-b49ebe2e0e3b.PNG)
